game_api.txt

```diff
@@ -206,6 +219,8 @@ The doors mod allows modders to register custom doors and trapdoors.
 	sound_open = sound play for open door, -- optional
 	sound_close = sound play for close door, -- optional
 	protected = false, -- If true, only placer can open the door (locked for others)
+	on_rightclick = function(pos, node, clicker, itemstack, pointed_thing) 
+	-- function containing the on_rightclick callback
```

Comment:
Needs a notice that it's optional - or - that it _defaults_ to a `doors.door_toggle`-wrapper function.